components/shared/carousel/RhCarousel.tsx

```diff
@@ -0,0 +1,52 @@
+import { css, StyleSheet } from "aphrodite";
+import { ReactElement, useState } from "react";
+import RhCarouselItem, { RhCarouselItemProps } from "./RhCarouselItem";
+
+type Props = {
+  rhCarouselItemProps: RhCarouselItemProps[];
+};
+
+const RhCarouselControl = ({
```

Comment:
I think this component should receive a list of carousel objects to render to make it more generic. At the moment it appears to be specific to a constant defined outside of component.

Reply:
Carousel renders ReactNodes in next Pr. Temp placeholder here :)